8_class_components/src/components/MemberDetails.js

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import company_logo from "../assets/company.png";
+import location_logo from "../assets/location.png";
+
+const Member = () => {
+	let { username } = useParams();
+	const [member, setMember] = useState("");
+
+	if (!username) return;
+
+	useEffect(() => {
+		getMemberDetails();
+	}, []);
```

Comment:
add respective dependencies that are involved.

Reply:
No dependencies to be specified in this case, I guess. Correct me if I am wrong.